### PR TITLE
K8s cleanup

### PR DESCRIPTION
**Pull request by ilausuch**
https://progress.opensuse.org/issues/120660

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ COPY requirements.txt /pcw/
 # * Install pip requirements
 # * Empty system cache to conserve some space
 RUN zypper -n in python310-devel gcc libffi-devel && pip3.10 install -r /pcw/requirements.txt && rm -rf /var/cache
-
+RUN zypper in -y kubernetes1.23-client
```
Comment (asmorodskyi) [inline Dockerfile:14]: just include this in list above , this will decrease build time

File: ocw/lib/EC2.py [hunk 2]
```diff
@@ -411,3 +412,69 @@ def cleanup_images(self):
...
+
+                    config_file = f"/tmp/k8s_{region}_{cluster_name}.yaml"
+
+                    # Write in YAML.
+                    config_text=yaml.dump(cluster_config, default_flow_style=False)
+                    open(config_file, "w").write(config_text)
+
+                    self.log_info(f"Created config file {config_file}")
+
+                    resp = subprocess.call(["kubectl", "--kubeconfig=%s" % config_file, "get", "nodes"])
```
Comment (asmorodskyi) [inline ocw/lib/EC2.py:475]: maybe you look for some python lib for interacting with k8s ?

Comment (asmorodskyi) [reply ocw/lib/EC2.py:475]: something like this https://github.com/kubernetes-client/python ? But please do better research it is just first URL from google

File: Dockerfile (hunk 1 above)
Comment (pdostal) [reply Dockerfile:14]: Build time and number of layers :smile:

File: .gitignore [hunk 3]
```diff
@@ -17,3 +17,6 @@ htmlcov/*
 # Configuration and program data
 /db
 pcw.ini
+
+# Development
+run/
```
Comment (asmorodskyi) [inline .gitignore:22]: we don't need this . normally you running things inside container

**Pull request closed without merging**